assets/check

```diff
@@ -84,16 +84,26 @@ lines_including_and_after() {
   sed -ne "/$escaped_string/,$ p"
 }
 
+get_commit(){
+  for TAG in $*; do
+    COMMIT=$(git rev-list -n 1 $TAG)
+    echo "{ \"ref\": \"$TAG\", \"commit\": \"$COMMIT\" }"
```

Comment:
I would suggest using `jq` to build the JSON using `--arg`, just in case someone has funny tag names:

```sh
jq -n '{ref: $tag, commit: $commit}' --arg tag $TAG --arg commit $COMMIT
```

Minor nit: we've been using lowercase for variable names since it's a bit harder to confuse them with global env vars.

Reply:
Thanks @vito
I updated the PR regarding using `jq` and lowercase variable

Reply:
Thanks @vito and @talset